x-pack/plugins/cases/public/components/configure_cases/connectors_dropdown.test.tsx

```diff
@@ -267,14 +99,34 @@ describe('ConnectorsDropdown', () => {
     ).not.toThrowError();
   });
 
-  test('it shows the deprecated tooltip when the connector is deprecated', () => {
+  it('shows the deprecated tooltip when the connector is deprecated', () => {
     render(<ConnectorsDropdown {...props} selectedConnector="servicenow-uses-table-api" />, {
       wrapper: ({ children }) => <TestProviders>{children}</TestProviders>,
     });
 
+    userEvent.click(screen.getByTestId('comboBoxToggleListButton'));
+
     const tooltips = screen.getAllByLabelText(
       'This connector is deprecated. Update it, or create a new one.'
     );
     expect(tooltips[0]).toBeInTheDocument();
   });
+
+  it('restores the previous connector if the user leaves the input empty', () => {
```

Comment:
The combobox allows the user to leave the input empty. I decided the beahviour for this case is to select the previously selected connector and if none was selected, select "none".